proposals/0250-raise-block-limits-to-60M.md

```diff
@@ -0,0 +1,88 @@
+---
+simd: '0256'
+title: Increase Block Limits to 60M CUs
+authors:
+  - Brennan Watt (Anza)
+category: Standard
+type: Core
+status: Review
+created: 2025-03-06
+feature: TBD
+development:
+  - Anza - TBD
+---
+
+## Summary
+
+Increase the block limit to 60M CUs.
+
+## Motivation
+
+Current block limits are set to 48M CUs. SIMD-0207 will increase them to 50M.
+Block limits' primary purpose is to ensure the vast majority of network
+participants are able to keep up with the network, by restricting the amount of
+work a leader is allowed to pack into a block. However, current mainnet traffic
+is largely not constrained by large block execution times. This proposal aims a
+substantial increase in block limits to 60M CUs, in order to provide additional
+capacity to the network.
+
+## New Terminology
+
+No new terms, but the following definitions are given for clarity:
+
+- Max Block Units - The maximum number of CUs that can be included in a block.
+- Max Writable Account Units - The maximum number of CUs that can be used to
+  write any given account in a block.
+- Max Vote Units - The maximum number of CUs that can be used by simple votes
+  in a block.
+- Max Block Accounts Data Size Delta - The maximum size of the accounts data
+  size delta in a block.
+
+## Detailed Design
```

Comment:
Is this proposal taking the place of #207 or intended to follow it? I assume follow it, and if that's the case we should make that explicitly clear in this section.

Reply:
Good callout. Intention is for this to follow #207 . I'll clarify in this section